<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeDataContainerInfoWidget</class>
 <widget class="QFrame" name="VolumeDataContainerInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>70</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Property</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dataContainerList"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="value">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>Resolution:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="resolutionLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>0.0, 0.0, 0.0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="value_3">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>Voxel Extents</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="voxelExtentsLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-weight: bold;
font-size: 10px;
}</string>
       </property>
       <property name="text">
        <string>100, 100, 100</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
